packages/amplication-server/src/core/workspace/workspace.service.ts

```diff
@@ -798,6 +798,55 @@ export class WorkspaceService {
     }
   }
 
+  async bulkUpdateWorkspaceProjectsAndResourcesLicensed(
+    useUserLastActive: boolean
+  ): Promise<boolean> {
+    try {
+      const date = new Date();
+      const userLastActiveQuery = useUserLastActive
+        ? {
+            some: {
+              lastActive: {
+                gte: new Date(date.setDate(date.getDate() - 30)),
```

Comment:
Would it make sense to make this (`30`) parameter customisable as environment variable?

Reply:
https://github.com/amplication/amplication-cluster-configuration/pull/157